spec/imsc-hrm.html

```diff
@@ -816,10 +819,10 @@ <h2>Paint Text</h2>
       typographical glyph aspect ratio. An implementation can determine an actual buffer size needs based on worst-case glyph size
       complexity.</p>
 
-      <p>The contents of the Glyph Buffer G<sub>n</sub> SHALL be copied instantaneously to Glyph Buffer G<sub>n-1</sub> at the
+      <p>The contents of the Glyph Buffer G<sub>n</sub> is copied instantaneously to Glyph Buffer G<sub>n-1</sub> at the
       presentation time of <a>Intermediate Synchronic Document</a> E<sub>n</sub>.</p>
 
-      <p>It SHALL be an <a>error</a> for the sum of NRGA(g<sub>i</sub>) over all <a data-lt="glyph">glyphs</a> Glyph Buffer G<sub>n</sub>
+      <p>It is an <a>error</a> for the sum of NRGA(g<sub>i</sub>) over all <a data-lt="glyph">glyphs</a> Glyph Buffer G<sub>n</sub>
```

Comment:
normative requirement for no error -> the SHALL is appropriate here?

Reply:
See https://github.com/w3c/imsc-hrm/pull/60#discussion_r1050046140